00_Base/src/interfaces/modules/AbstractModule.ts

```diff
@@ -144,6 +155,17 @@ export abstract class AbstractModule implements IModule {
         }
     }
 
+    async handleMessageApiCallback(message: IMessage<OcppResponse>): Promise<void> {
+        const url: string | null = await this._cache.getAndRemove(message.context.correlationId, this.CALLBACK_URL_CACHE_PREFIX + message.context.stationId);
+        if (url) {
+            try {
+                await axios.post(url, message.payload);
+            } catch (error) {
+                this._logger.error("Failed sending call result: ", error);
```

Comment:
@thanaParis Ideally the error log is also stored in the database in a failed invocations table to ensure these are visible outside of a log file.

Reply:
This is currently impossible, given our persistence logic is in 10_Data which relies on 00_Base. However, I've added a comment to reflect this point.